src/main/java/kata/makers/Checkout.java

```diff
@@ -0,0 +1,31 @@
+package kata.makers;
+
+import java.util.Arrays;
+
+public class Checkout {
+
+  public int calculateSum(String numbersString) throws Exception {
+
+    if (numbersString.contains(",")) {
+      return calculateSumForCommaSeparatedNumbers(numbersString);
+    } else if (numbersString.contains("\n")) {
+      return calculateSumForNewLineSeparatedNumbers(numbersString);
+    } else {
+      throw new InvalidCharacterException("Cannot accept single character");
+    }
+  }
+
+  private static int calculateSumForCommaSeparatedNumbers(String numbersString) throws Exception {
+    if (numbersString.endsWith(",")) {
+      throw new InvalidCharacterException("Cannot end with comma");
+    }
+    return Arrays.stream(numbersString.split(",")).mapToInt(Integer::valueOf).sum();
+  }
+
+  private static int calculateSumForNewLineSeparatedNumbers(String numbersString) throws Exception {
+    if (numbersString.endsWith("\n")) {
+      throw new InvalidCharacterException("Cannot end with new line");
+    }
```

Comment:
Logic present in both the methods look alike. Is it possible to generify the methods for both the characters? 
In future, if we have the different input characters, it would be better to handle with a generic method